$NetBSD: patch-ba,v 1.2 2009/11/29 20:40:08 sno Exp $

replacing explicit --export-dynamic by ${EXPORT_SYMBOLS_LDFLAGS}
for non-gnu-ld

--- configure.orig	2009-06-27 18:32:33.000000000 +0000
+++ configure	2009-11-29 20:29:57.000000000 +0000
@@ -16179,7 +16179,7 @@
 
     if ".$can_build_shared" = .no
     then ZZIPLIB_LDFLAGS=""
-    else ZZIPLIB_LDFLAGS="--export-dynamic"
+    else ZZIPLIB_LDFLAGS="${EXPORT_SYMBOLS_LDFLAGS}"
     fi
     RESOLVES=' # '
   ;;
